tests/installation/partitioning/suggested_partitioning/verify_decrypted_partition_deleted.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The test module reuses the partition that was created with LVM on
```

Comment:
We could update the summary and delete all the comment inside `run` (it is just my personal taste that it is distracting to have a comment which is obvious). Saying that we will use two list it is clear that the size should be same or that the module would die if any error. I would also improve the error in line 35 to avoid any comment on top, something like "expected entry '$foo' not found in corresponding block '$bar' from Suggested Partitioner summary.
(optional, really matter of taste)